go.mod

```diff
@@ -58,20 +70,47 @@ require (
 	github.com/modern-go/reflect2 v1.0.2 // indirect
 	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822 // indirect
 	github.com/mwitkow/go-conntrack v0.0.0-20190716064945-2f068394615f // indirect
+	github.com/pkg/profile v1.3.0 // indirect
 	github.com/prometheus/procfs v0.10.1 // indirect
+	github.com/robfig/cron v1.2.0 // indirect
 	github.com/sirupsen/logrus v1.9.0 // indirect
 	github.com/spf13/pflag v1.0.5 // indirect
+	github.com/stoewer/go-strcase v1.2.0 // indirect
+	go.etcd.io/etcd/api/v3 v3.5.10 // indirect
```

Comment:
Seems like there's maybe room for improvement in how library-go structures packages, because the new controller cannot possibly depend on etcd APIs.  And yet:

```console
membershippb/ mvccpb/       v3rpc/        version/      
wking@penguin:~/src/cluster-version-operator$ ls vendor/go.etcd.io/etcd/api/v3/authpb/auth.p
auth.pb.go  auth.proto  
wking@penguin:~/src/cluster-version-operator$ go mod why go.etcd.io/etcd/api/v3/version
# go.etcd.io/etcd/api/v3/version
github.com/openshift/cluster-version-operator/cmd
github.com/openshift/library-go/pkg/controller/controllercmd
github.com/openshift/library-go/pkg/config/serving
k8s.io/apiserver/pkg/server/options
k8s.io/apiserver/pkg/storage/storagebackend/factory
go.etcd.io/etcd/client/v3
go.etcd.io/etcd/api/v3/version
```

But they're OCP dependencies, so dragging them over to this repo's vendor directory doesn't extend our product-scoped exposure.